src/psyclone/psyGen.py

```diff
@@ -2711,6 +2725,14 @@ def _find_subclasses(self, module, base_class):
                 issubclass(cls, base_class) and cls is not base_class]
 
 
+@dataclass
+class ValidOption:
+    '''Class used to specify the valid options dict for a Transformation.'''
+    default: object
```

Comment:
It's 'early' and I've not had much coffee but I'm struggling to remember what these different elements correspond to. Please could you document them (using the normal :param syntax in the docstring).

Reply:
Added short explanations - if you think they need expanding let me know but I think they're fairly straightforward.